pkgs/applications/networking/browsers/chromium/source/sources.nix

```diff
@@ -1,21 +1,17 @@
 # This file is autogenerated from update.sh in the parent directory.
 {
   dev = {
-    version = "49.0.2593.0";
-    sha256 = "1krkf8dg10xnvxs4zfvy2rkp0v3gxg55418b11kmfywsxjfi09hv";
-    sha256bin32 = "0sarfj8r6x28pbmcpd5facm6liifazj744c9qn6ci6xg7f8jlnxi";
-    sha256bin64 = "1l5s504mmr4v5y67fsla9hyrk7dm155yc790ld87s1ya03h455gr";
+    version = "50.0.2657.0";
+    sha256 = "1xc66zsjd5snydf60di2k2vzwwcqs1p16yn4jbpj2khm9cmxmkmf";
```

Comment:
The `sha256bin64` is missing here and for beta. @aszlig added two to add those here: 716b79d3a5f3ee438942f59bfd879f46e66204ac & 54b49125664adb1f7cf054bbf8a09aeecc88b574. Maybe backport those as well?